frontend/src/components/wrappers/MetricsVisualizations.tsx

```diff
@@ -0,0 +1,291 @@
+/*
```

Comment:
Sounds good on moving this file to `viewers` folder. For now I want to keep the code in this file staying in this file. Because there are many shared logic which only makes sense for Visualization Tab V2. So it becomes harder to manage if we put variants in multiple files. The best time to separate variants to corresponding files is when we deprecate Visualization Tab v1, so the complexity of codebase is well controlled.